legal-api/src/legal_api/core/filing.py

```diff
@@ -79,6 +79,7 @@ class FilingTypes(str, Enum):
         SPECIALRESOLUTION = 'specialResolution'
         TRANSITION = 'transition'
         PUTBACKON = 'putBackOn'
+        RESTORATION = 'restoration'
```

Comment:
Could you put this in alpha order, and fix the putBackOn too?
It'll make the longterm maintenance and debugging easier